plugins/jobs/src/js/JobsMenu.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-unused-vars */
+import React from "react";
+/* eslint-enable no-unused-vars */
+
+import { componentFromStream } from "data-service";
+
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/do";
+import "rxjs/observable/empty";
+
+import Page from "#SRC/js/components/Page";
+import JobsBreadcrumbs from "#SRC/js/components/breadcrumbs/JobsBreadcrumbs";
+
+import { jobsRunNowAction, jobsRunNow$ } from "./JobsRunNow";
+
+const getActions = function(props) {
+  const [edit, ...rest] = props.actions;
+  const newActions = [];
+
+  newActions.push(edit);
+
+  newActions.push(jobsRunNowAction(props.job.getId()));
+
+  return newActions.concat(rest);
+};
+
+const JobsMenu = componentFromStream(props$ => {
```

Comment:
What about snapshot testing this one? We can have the cases where it has the schedule or not.

Reply:
Also, in the test with the snapshot the componentFromStream doesn't initialize correctly. Will need some investigation.